### ISSUE TITLE
[high] Color Contrast: Set password: Green text over light grey background fails minimum contrast requirements

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5526]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the 'Enter your phone or email' field and enter a valid email address, then click on 'Continue' button.
3. Open the received email and click on 'here' link.
4. Once the 'Set password' page opens, enter a password matching the requirements into 'Enter a password' text field.
5. Measure the color contrast ratio of the green 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, 1 number' text against light grey background.

## Expected Result:
The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).

## Actual Result:
The green text (#03D484) against light grey background (#FAFAFA) has a color contrast ratio of 1.87:1. The text is small (13px=9.75pt).

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(3, 212, 124); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px; margin-top: 4px; margin-bottom: 4px;">Your password must have at least 8 characters,
1 capital letter, 1 lowercase letter, 1 number.</div>`

CSS:
`element.style {
color: rgb(3, 212, 124);
font-family: GTAmericaExp-Regular;
font-size: 13px;
line-height: 18px;
margin-top: 4px;
margin-bottom: 4px;
}`

## Workaround:
No workaround

## Area issue was found in:
Set password page

## Failed WCAG checkpoints
1.4.3

## User impact:
The low-vision and color-blind users might not be able to see the text.

## Suggested resolution:
Use a darker shade of green for text to achieve a color contrast ratio of at least 4.50:1. One example of color to use is `#038251`, which will give a color contrast ratio of 4.66:1.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5183578_color_contrast_-_green_text](https://user-images.githubusercontent.com/88733897/134780146-f7b792c5-3d9f-4935-be32-eb58937556ed.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
I believe we are going to be removing the green text for the password hint. Does that sound familiar @anthony-hull?

**Comment by anthony-hull** [comment]
Yes the green styling has been removed in main

**Comment by shawnborton** [comment]
Do you know when that change went live @anthony-hull? Perhaps we can retest and then potentially close out this issue and focus on other accessibility issues.

**Comment by anthony-hull** [comment]
![image](https://user-images.githubusercontent.com/47184118/134892266-8c2b254b-3ba0-42ab-9dec-9239b4155225.png)

**Comment by shawnborton** [comment]
@ogumen do you mind testing this again?

**Comment by anthony-hull** [comment]
here is the commit in github:
https://github.com/Expensify/App/commit/f4117ee9539098ba53a39d3e953b4d217e9d11ea

**Comment by Luke9389** [comment]
We've changed this to use the `color.red` which is `#FC3826`. The background is `#F9F9F9`

This is still only a contrast ratio of 3.47:1 - still a fail for normal text but a pass for large text.

**Comment by Luke9389** [comment]
<img width="1848" alt="Screen Shot 2021-09-30 at 4 21 36 PM" src="https://user-images.githubusercontent.com/49007721/135555702-714962e4-194e-4f44-bc99-b70f59c9ba79.png">

**Comment by shawnborton** [comment]
Yup - I think we'll need a darker red for this, but perhaps we should put this on hold until we dive into colors with Alto.

**Comment by ogumen** [comment]
> 
> 
> @ogumen do you mind testing this again?

@shawnborton the color of the password requirements text when the password requirements match is `#7D8B8F`, while the background is same `#FAFAFA`, resulting in a color contrast ratio of 3.37:1. The current ratio still fails the minimum contrast requirements for small text (should be at least 4.50:1).
![Password requirements match](https://user-images.githubusercontent.com/88733897/135860184-309e03c8-ea25-4bca-895f-001b1f1ea900.png)
